webapp/templates/pokemon-table.component.html

```diff
@@ -1,10 +1,73 @@
-<table>
-	<tr>
-		<th *ngFor="let stat of _pokemonTableStats" (click)="_sortPokemon(stat.property, true)">{{stat.heading}}</th>
-		<th *ngIf="_showTransferColumn">Transfer</th>
-	</tr>
-	<tr *ngFor="let mon of _pokemon; let i = index">
-		<td *ngFor="let stat of _pokemonTableStats"><span *ngIf="_typeof(mon[stat.property]) === 'boolean' && mon[stat.property]">✓</span><span *ngIf="_typeof(mon[stat.property]) !== 'boolean'">{{mon[stat.property]}}</span></td>
-		<td *ngIf="_showTransferColumn"><button type="button" [disabled]="mon.favorite || _transferringPokemonAtIndex !== null" (click)="_transferPokemon(mon, i)">{{_getTransferButtonText(i)}}</button></td>
-	</tr>
-</table>
\ No newline at end of file
+<div class="sortby-wrapper">
+	<div class="sortby-item" *ngFor="let stat of _pokemonTableStats" (click)="_sortPokemon(stat.property, true)">
+		{{stat.heading}}
+	</div>
+</div>
+
+<div class="pokemon-list-wrapper">
+  <div class="pokemon-list-item" *ngFor="let mon of _pokemon">
```

Comment:
I think you need to change the *ngFor to be `*ngFor="let mon of _pokemon; let i = index"` so we don't break transferring - and so that renaming will work. Unless you're doing that somewhere that I missed?